client/core/notification.go

```diff
@@ -761,6 +765,8 @@ func newUnknownBondTierZeroNote(subject, details string) *db.Notification {
 const (
 	ActionIDRedeemRejected = "redeemRejected"
 	TopicRedeemRejected    = "RedeemRejected"
+	ActionIDRefundRejected = "refundRejected"
+	TopicRefundRejected    = "RefundRejected"
```

Comment:
In the UI the action was updated to `txRejected`, so when these actions are sent, there is a UI error.

Reply:
Oh yeah, thanks. I guess keeping two here as `handleCoreAction` needs them to be different. Reverting the js change and adding `refundRejected`